apps/web/src/services/asset-inventory/components/ServiceAccountDeleteModal.vue

```diff
@@ -50,10 +52,14 @@ const state = reactive({
 /* Api */
 const deleteServiceAccount = async () => {
     try {
-        if (state.isGeneralAccount && ('service_account_id' in props.serviceAccountData)) {
+        if (state.isGeneralAccount && props.serviceAccountData && ('service_account_id' in props.serviceAccountData)) {
```

Comment:
suggestion: How about
```typescript
'service_account_id' in props.serviceAccountData

// tobe
props.serviceAccountData?.service_account_id
```

Reply:
This code had to be written as above for TypeScript errors.